leveler/image_generators.py

```diff
@@ -229,31 +236,30 @@ def _write_unicode(text, init_x, y, font, unicode_font, fill):
         )  # Symbol
 
         # userinfo
-        server_rank = "#{}".format(server_rank)
+        server_rank = "#{}".format(self._humanize_number(server_rank))
         draw.text(
             (self._center(100, 200, server_rank, large_fnt), v_label_align - 30),
             server_rank,
             font=large_fnt,
             fill=info_text_color,
         )  # Rank
-        level_text = "{}".format(userinfo["servers"][str(server.id)]["level"])
+        level_text = "{}".format(
+            self._humanize_number(userinfo["servers"][str(server.id)]["level"])
+        )
         draw.text(
             (self._center(95, 360, level_text, large_fnt), v_label_align - 30),
             level_text,
             font=large_fnt,
             fill=info_text_color,
         )  # Level
-        credit_txt = (
-            f"{bank_credits}"
-            f"{credits_name if (credits_name := (credits_name)[0]) != '<' else '$'}"
-        )
+        credit_txt = f"{self._humanize_number(bank_credits)}"
         draw.text(
             (self._center(260, 360, credit_txt, large_fnt), v_label_align - 30),
             credit_txt,
             font=large_fnt,
             fill=info_text_color,
         )  # Balance
-        exp_text = f"{exp_frac}/{exp_total}"
+        exp_text = f"{self._humanize_number(exp_frac)}/{self._humanize_number(exp_total)}"
```

Comment:
Noted, although it might be a good idea to add some sort of function to add human readability. As levels get higher and higher, the number becomes increasingly difficult to read. Consider using the [`redbot.core.utils.chat_formatting.humanize_number`](redbot.core.utils.chat_formatting.humanize_number) method to make the numbers a bit more manageable (i.e., 12345 -> 12,345 or 12.345 depending on locale)